### ISSUE TITLE
How much time does it take to test A11y?

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#222]
After a discussion in an a11y meeting today, it would be helpful to understand how much time it takes to test A11y features. This was motivated by a discussion regarding whether or not it is worthwhile to test a11y in the LoL harness. @KatieWoe could you please roughly describe how much time or additional effort it takes for you/the QA team to test a11y in various areas like dev tests/iframe tests/LoL harness tests, and so on? This will help us understand how much time it takes you and the QA team to do these things when we make testing tasks.

Also assigning to @emily-phet to track and add any clarifications.

Thanks!

**Comment by KatieWoe** [comment]
Likely it would not take much additional time, but it depends on how buggy it actually is.  When testing the sim itself, a11y testing takes about the same amount of time as a normal test on that platform/browser (40 min to an hour, though that does depend on how many bugs are found).  If testing in, say, iframe though, it would likely be a quick check to make sure a few things were being read and that it didn't behave differently from the normal sim.

**Comment by KatieWoe** [comment]
That being said, it would be hard to predict and I may be off here.

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe, thats helpful. What about LoL, does that fall into the iframe category of time it takes, or does a11y testing in LoL double the time it takes to test LoL features overall?

**Comment by KatieWoe** [comment]
LoL usually doesn't take much time. a11y probably wouldn't add much, but set up might depending on when people did it.

**Comment by emily-phet** [comment]
Would it be possible, without incurring significant time/trouble, to actually time the testing process for each step in testing the a11y features for either the next RC release of RIAW, or the next sim to get a first RC? I ask because it might be nice to check in every few sims or so on the actual time it takes, so that everyone (across dev, design, QA, etc.) can have a concrete sense of what's 'typical', though certainly each sim will vary. 

It may also be useful when indicating the different costs incurred when including a11y features into interactives to be able to point some specific sim examples and talk about the sim design, dev, and QA process.

**Comment by KatieWoe** [comment]
I could do that for a few browsers/platforms.  The difficulty would be in the fact that multiple people are doing the tests and I'm often splitting my attention between testing and other QA tasks, lengthening the time.

**Comment by emily-phet** [comment]
Just some ballpark numbers would be good - doesn't have to be exact.

**Comment by KatieWoe** [comment]
I did this during https://github.com/phetsims/QA/issues/276 for jaws and nvda.  I didn't really see any differences in terms of how the screen readers behaved.  I'd say it took about 15 extra minutes on nvda and about 30+ for jaws.
@emily-phet @jessegreenberg

**Comment by lmulhall-phet** [comment]
Should we close this issue?

**Comment by KatieWoe** [comment]
That's up to @jessegreenberg and @emily-phet
